spid_cie_oidc/provider/settings.py

```diff
@@ -0,0 +1,47 @@
+
+from spid_cie_oidc.entity.utils import exp_from_now, iat_now
+
+JWT_CLIENT_ASSERTION = {
+    "iss": "http://rp-test/oidc/rp",
+    "sub": "http://rp-test/oidc/rp",
+    "iat": iat_now(),
```

Comment:
I still don't understand the meaning of this file, can you explain why you created it @francescatronconi

Reply:
I have removed this with the commit[1855b52], I don't understand why you still see them